manifests/enterprise/dashboard/config.pp

```diff
@@ -0,0 +1,18 @@
+class sensu::enterprise::dashboard::config {
+  if $caller_module_name != $module_name {
+    fail("Use of private class ${name} by ${caller_module_name}")
+  }
+
+  if $sensu::purge_config and !$sensu::enterprise_dashboard {
```

Comment:
Likely #401 will end up being merged which changes the purge options (just a heads up)